### PR TITLE
Create IUFRO-HQ.csl

### PR DESCRIPTION
**Pull request by IUFRO**
Style designed to be used by authors publishing with the International Union of Forest Research Organizations, but open to be used by anyone looking for a simple and consistent style

### CODE REVIEW

**Comment by rmzelle** [comment]
Thanks for your submission. Is there an online style guide available for this style? Per https://github.com/citation-style-language/styles/wiki/Style-Requirements#10---documentation-link, we typically require each CSL style to include a link to the format it describes.

File: IUFRO-HQ.csl [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+      <key macro="author" names-min="3" names-use-first="1"/>
+      <key macro="issued"/>
+    </sort>
+    <layout suffix=".">
+      <group delimiter=" ">
+        <text macro="author"/>
+        <text macro="issued" prefix="(" suffix=")."/>
+        <choose>
+          <if type="article article-magazine article-newspaper article-journal review manuscript paper-conference" match="any">
+            <text variable="title" text-case="sentence" font-style="italic" suffix="."/>
```
Comment (adam3smith) [inline IUFRO-HQ.csl:78]: We don't use `text-case="sentence"` for titles and it will be deprecated in the future (as it can't handle proper names). If you want titles in sentence case, just don't use a `text-case` attribute. You should assume titles are stored in sentence case in the metadata.

File: IUFRO-HQ.csl [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  <macro name="publisher">
+    <group delimiter=": ">
+      <text variable="publisher"/>
+      <text variable="publisher-place" suffix="."/>
+    </group>
+  </macro>
+  <macro name="editor">
+    <names variable="editor" suffix=".">
+      <label form="short" text-case="capitalize-first" prefix="(" suffix=") "/>
+      <name and="text" et-al-min="5" initialize-with="."/>
```
Comment (adam3smith) [inline IUFRO-HQ.csl:47]: please always use `et-al-min` and `et-al-use-first` jointly to avoid confusion. Inheriting one from a higher level is super hard to follow in a style.

File: IUFRO-HQ.csl [hunk 3]
```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" et-al-min="3" initialize-with="." page-range-format="expanded" demote-non-dropping-particle="sort-only" default-locale="en-GB">
```
Comment (adam3smith) [inline IUFRO-HQ.csl:2]: I'd generally avoid setting `et-al-min` at the style level, especially since it looks like you're overriding it for every occurrence.

File: IUFRO-HQ.csl [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>International Union of Forest Research Organizations - Headquarters</title>
+    <title-short>IUFRO - HQ</title-short>
+    <id>http://www.zotero.org/styles/international-union-of-forest-research-organizations-headquarters</id>
```
Comment (adam3smith) [inline IUFRO-HQ.csl:7]: the filename should be the same as the id (plus the `.csl` extension of course)

**Review by adam3smith — CHANGES_REQUESTED** [review]
Some comments on the style -- also see Rintze's question about the style guide

File: IUFRO-HQ.csl (hunk 1 above)
Comment (IUFRO) [reply IUFRO-HQ.csl:78]: Thank you, I was not aware of this issue. I removed the 'text-case' attribute in the new version.

File: IUFRO-HQ.csl (hunk 2 above)
Comment (IUFRO) [reply IUFRO-HQ.csl:47]: Following this comment, I added 'et-al-use-first' in all instances where I used 'et-al-min'

File: IUFRO-HQ.csl (hunk 3 above)
Comment (IUFRO) [reply IUFRO-HQ.csl:2]: I removed 'et-al-min' at the style level

File: IUFRO-HQ.csl (hunk 4 above)
Comment (IUFRO) [reply IUFRO-HQ.csl:7]: I changed the name of the file to match the id.

**Comment by IUFRO** [comment]
Thank you both for your informative comments. Please find the new version with your comments addressed in the 'Commits' section.

**Comment by rmzelle** [comment]
> Thank you both for your informative comments. Please find the new version with your comments addressed in the 'Commits' section.

Thanks, almost there.

Did you base this on an existing CSL style? If so, we'd like to know which one, so we can add a "template" link (see https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link)

**Comment by IUFRO** [comment]
> 
> 
> > Thank you both for your informative comments. Please find the new version with your comments addressed in the 'Commits' section.
> 
> Thanks, almost there.
> 
> Did you base this on an existing CSL style? If so, we'd like to know which one, so we can add a "template" link (see https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link)

Thanks! No, I did not use an existing CSL style. I opened the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/), removed the example that appeared by default, and started filling in the necessary information until I got the style in the format I wanted.

I hope that helps!

**Comment by rmzelle** [comment]
Okay, thanks!
